libs/application/templates/parental-leave/src/lib/dataSchema.ts

```diff
@@ -51,7 +51,11 @@ export const dataSchema = z.object({
   }),
   shareInformationWithOtherParent: z.enum([YES, NO]),
   usePrivatePensionFund: z.enum([YES, NO]),
-  employerInformation: z.object({ email: z.string().email() }).optional(),
+  employerNationalRegistryId: z
+    .string()
+    .refine((n) => n && kennitala.isValid(n) && kennitala.isPerson(n), {
+      params: errorMessages.otherParentId,
```

Comment:
This message contains a generic `invalid kennitala`. I was wondering if I should create a duplicate to use here but went with the other one.

Reply:
I'll create a duplicate just in case.